src/CMakeLists.txt

```diff
@@ -466,6 +466,20 @@ if(WIN32)
   # The UNICODE and _UNICODE definitions are needed to make
   # (QString) mudlet::getShortPathName(const QString& name) work in Windows:
   target_compile_definitions(mudlet PRIVATE UNICODE _UNICODE)
+
+  add_custom_command(TARGET mudlet POST_BUILD
+          COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_RUNTIME_DLLS:mudlet> $<TARGET_FILE_DIR:mudlet>
```

Comment:
Minimum CMake version needs a bump due to this, as well as the version of cmake that functions.ps1 install